### ISSUE TITLE
CommandBar split buttons not accessible to screenreader users, axe-core errors

### ISSUE THREAD

**Issue by BrendanMcK** [microsoft/fluentui#16584]
### Environment Information

- **Package version(s)**: 7.156
- **Browser and OS versions**: n/a

### Describe the issue:

Couple of issues: first, we're seeing failures with CommandBar split buttons when using eg Accessibility Insights, due to incorrect roles - can repro this on the [Fluent command bar page](https://developer.microsoft.com/en-us/fluentui#/controls/web/commandbar) ([Permalink version for 7.156.0](https://fabricweb.z5.web.core.windows.net/oufr/7.156.0/#/examples/commandbar)):

![image](https://user-images.githubusercontent.com/1754373/105460265-65c4f100-5c40-11eb-87e9-96b5c4d614c1.png)

Looks like the issue is with the aria roles used in the tree:

![image](https://user-images.githubusercontent.com/1754373/105460662-fe5b7100-5c40-11eb-9d80-d3e18ed1d22e.png)

There's a `div` with `role=menuitem` - which is fine; but it in turn contains a `button` with `role=menuitem`, and another `button` without any role (so would default to `role=button`). Neither of these are allowable children within the parent `role=menuitem`. (The `span` might also be causing an issue here, as it's lacking `role=none` or `role=presentation`)

--

There's a related issue here re usability for screenreader users: there's only a single keyboard focusable element here, so keyboard users have to tab to the overall combo item, and use Enter to activate the default action, or Alt+Down to activate the popup.

However, this information is _only conveyed visually_ via the line splitting the button text from the chevron:

Regular single-action menu button:
![image](https://user-images.githubusercontent.com/1754373/105461604-521a8a00-5c42-11eb-8210-ca1959e61c8f.png)
Split button with single focus but two keyboard actions:
![image](https://user-images.githubusercontent.com/1754373/105461684-69597780-5c42-11eb-8e69-8706ab759473.png)

These use the same aria roles; so screenreader users are only aware that this is a button with a popup, so can't distinguish a split button from a regular button menu with a single action - so there's no way to discover the availability of the Alt+Down action.

Fixing this so that it's usable to screenreader users might require changing the keyboard interaction model, or reworking what roles are used here. I'm not sure that there's actually a 'blessed' approach in aria for this type of control, though.

--

Anyhow, despite appearing to be two independent issues, it may be best to address both of these holistically, since the aria structure impacts both.

(There is an [existing closed bug re keyboard accessibility](https://github.com/microsoft/fluentui/issues/15638) of these - however it was closed without taking screenreader users into account.)

--

**Comment by gouttierre** [comment]
@BrendanMcK - Thanks for the feedback and providing detailed information into the issue you are experiencing. 

@khmakoto  - Would you be able to determine if this is a regression or if this behavior is an issue? - Not sure if you will need to review the existing closed bug re keyboard [accessibility ](https://github.com/microsoft/fluentui/issues/15638) for more connect on the screen reader. 
cc @tomi-msft , @smhigley
